### PR TITLE
Handle a None deploy_group properly

### PR DESCRIPTION
**Pull request by solarkennedy**
This is prerequisite to doing local-run tron.

### CODE REVIEW

File: paasta_tools/cli/cmds/local_run.py [hunk 1]
```diff
@@ -866,17 +866,27 @@ def configure_and_run_docker_container(
...
-                PaastaColors.red(
-                    "Error: No sha has been marked for deployment for the %s deploy group.\n"
-                    "Please ensure this service has either run through a jenkins pipeline "
-                    "or paasta mark-for-deployment has been run for %s\n" % (
-                        instance_config.get_deploy_group(), service,
+            if instance_config.get_deploy_group() is None:
+                paasta_print(
+                    PaastaColors.red(
+                        f"Error: {service}.{instance} has no 'deploy_group' set. Please set one so "
+                        "the proper version of the code can be downloaded for use.",
```
Comment (mattmb) [inline paasta_tools/cli/cmds/local_run.py]: Wording sounds a bit weird to me. Maybe "Please set one so PaaSTA knows which image to run for this service" is better?

**Comment by mattmb** [comment]
Is it worth improving the errors anywhere else we have that Exception?

File: paasta_tools/cli/cmds/local_run.py (hunk 1 above)
Comment (solarkennedy) [reply paasta_tools/cli/cmds/local_run.py]: fixed

**Comment by solarkennedy** [comment]
chronos-rerun is the only other "user-facing" command where this could happen, but it is much less likely. Meh.

**Pull request merged**